client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/translations.tsx

```diff
@@ -3,24 +3,21 @@ import { translate } from 'i18n-calypso';
 import { TranslatedLaunchpadStrings } from './types';
 
 export function getLaunchpadTranslations( flow: string | null ): TranslatedLaunchpadStrings {
-	const translatedStrings = {
+	const translatedStrings: TranslatedLaunchpadStrings = {
 		flowName: translate( 'WordPress' ),
-		sidebarTitle: translate( 'Your website is ready to launch!' ),
-		sidebarSubtitle: translate( 'Keep up the momentum with these final steps.' ),
+		title: translate( 'Your website is ready!' ),
+		subtitle: translate( 'Keep up the momentum with these final steps.' ),
 	};
 
 	switch ( flow ) {
 		case NEWSLETTER_FLOW:
 			translatedStrings.flowName = translate( 'Newsletter' );
-			translatedStrings.sidebarTitle = translate( 'Your Newsletter is ready to launch!' );
+			translatedStrings.title = translate( 'Your Newsletter is ready!' );
 			break;
 		case LINK_IN_BIO_FLOW:
 			translatedStrings.flowName = translate( 'Link in Bio' );
-			translatedStrings.sidebarTitle = translate( 'Your Link in Bio is ready to launch!' );
-			break;
-		case 'podcast':
-			translatedStrings.flowName = translate( 'Podcast' );
-			translatedStrings.sidebarTitle = translate( 'Your Podcast is ready to launch!' );
+			translatedStrings.title = translate( 'Your Link in Bio is almost ready!' );
+			translatedStrings.launchTitle = translate( 'Your Link in Bio is ready to launch!' );
```

Comment:
Can we add specs for this?

Reply:
Great idea and will do. It looks like there are already launchpad translation tests that will break with this PR. So I'll update those and add any new ones that are needed. The fact that there are tests that will break suggests that, as we add more test coverage for Launchpad, I/we need to make a habit of running those tests when prepping our PRs.

Reply:
@jeyip - Translation tests have been updated/added.